services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from db.models import Order, Ticket
+
+
+def create_order(tickets, username, date=None):
+    with transaction.atomic():
+        created_order = Order.objects.create(
+            user=get_user_model().objects.get(username=username)
+        )
+
+        if date:
+            created_order.created_at = date
+
+        created_order.save()
+
+        for ticket in tickets:
+            row, seat, movie_session = ticket.values()
```

Comment:
I'm not sure that it is good practise. What if the order of dictionary ticket.values() will change?